packages/create-ponder/package.json

```diff
@@ -41,10 +41,9 @@
     "abitype": "^0.10.2",
     "dotenv": "^16.3.1",
     "tsup": "^8.0.1",
-    "typescript": "^5.3.2",
```

Comment:
Is this because we're using the top-level monorepo typescript?

Reply:
Yes